packages/next/src/cli/next-dev.ts

```diff
@@ -200,7 +200,7 @@ const nextDev: CliCommand = async (argv) => {
       Log.warn(
         'Your project has `@next/font` installed as a dependency, please use the built-in `next/font` instead. ' +
           'The `@next/font` package will be removed in Next.js 14. ' +
-          'You can migrate by running the `built-in-next-font` codemod. Read more: https://nextjs.org/docs/messages/built-in-next-font'
+          'You can migrate by running `npx @next/codemod built-in-next-font`. Read more: https://nextjs.org/docs/messages/built-in-next-font'
```

Comment:
```suggestion
          'You can migrate by running `npx @next/codemod built-in-next-font .`. Read more: https://nextjs.org/docs/messages/built-in-next-font'
```

I think you need to specify the directory with a `.` right?

Reply:
You get a prompt to specify the path, but might as well add it 👍